python/plugins/processing/gui/ScriptEditorDialog.py

```diff
@@ -139,19 +138,10 @@ def __init__(self, algType, alg):
 
         self.editor.setLexerType(self.algType)
 
-    def increaseFontSize(self):
-        font = self.editor.defaultFont
-        self.editor.setFonts(font.pointSize() + 1)
-        self.editor.initLexer()
-
-    def decreaseFontSize(self):
-        font = self.editor.defaultFont
-        self.editor.setFonts(font.pointSize() - 1)
-        self.editor.initLexer()
 
     def showSnippets(self, evt):
         popupmenu = QMenu()
-        for name, snippet in list(self.snippets.items()):
+        for name, snippet in self.snippets.iteritems():
```

Comment:
In python 2 it's `iteritems()` in python 3 it's `items()` and for compatibility `list(items())` was there.